### DISCUSSION TITLE
Unity Game창에 아이폰을 연결해서 카메라를 비추는 방법?

### DISCUSSION THREAD

**Discussion by hemil0102** [hemil0102/XRStudy#50] [7. Q&A]
![Screenshot 2024-07-02 at 2 30 06 PM](https://github.com/hemil0102/XRStudy/assets/83139316/b24a1080-27db-4ddc-bf2c-65f420f1e0b5)

어떤 개발자의 개발환경을 보니 빌드를 하지 않고 게임만 실행해도 Game 창에 카메라 촬영이 되는 모습을 볼 수 있는데 어떻게 관련 설정을 할 수 있나요?

**Comment by jongfeel** [comment]
이건 Unity Remote라고 iphone에 앱을 설치하고 unity editor와 원격으로 연결해서 사용하는 방식입니다.
아래 unity manual 참고하시면 설정하실 수 있습니다.

https://docs.unity3d.com/6000.0/Documentation/Manual/UnityRemote5.html
